constants.gradle

```
@@ -1,10 +1,10 @@
 project.ext {
     ANDROID_MIN_SDK_VERSION = 16
-    ANDROID_TARGET_SDK_VERSION = 27
-    ANDROID_COMPILE_SDK_VERSION = 27
-    ANDROID_BUILD_TOOLS_VERSION = '27.0.3'
+    ANDROID_TARGET_SDK_VERSION = 28
+    ANDROID_COMPILE_SDK_VERSION = 28
+    ANDROID_BUILD_TOOLS_VERSION = '28.0.2'
     PLAY_SERVICES_VERSION = '11.0.4'
     GOOGLE_IMA_SDK_VERSION = '3.7.4'
-    APPCOMPAT_V7_VERSION = '27.0.0'
-    SUPPORT_V4_VERSION = '27.0.0'
+    APPCOMPAT_V7_VERSION = '27.1.1'
```

Since we are upgrading Android dependencies to v28, should we update APPCOMPAT_V7_VERSION to v28?
```suggestion
    APPCOMPAT_V7_VERSION = '28.0.0'
```